walk-in-interview/src/main/webapp/new-job.js

```diff
@@ -4,24 +4,214 @@
  * is submitted.
  */
 
+/**
+ * Titles for all the sections/fields in different languages.
+ */
+const NEW_JOB_PAGE_TITLE = {
+  en: 'New Job Post',
+};
+const NEW_JOB_CANCEL_TITLE = {
+  en: 'Cancel',
+};
+const NEW_JOB_SUBMIT_TITLE = {
+  en: 'Create',
+};
+const NEW_JOB_TITLE = {
+  en: 'Job Title',
+};
+const NEW_JOB_DESCRIPTION = {
+  en: 'Describe the job here (responsibilities, preffered skills)...',
+};
+const NEW_JOB_ADDRESS = {
+  en: 'Job Address',
+};
+const NEW_JOB_REQUIREMENTS_TITLE = {
+  en: 'Requirements',
+};
+const NEW_JOB_PAY_TITLE = {
+  en: 'Job Pay',
+};
+const NEW_JOB_PAY_MIN_TITLE = {
+  en: 'min (sgd)',
+};
+const NEW_JOB_PAY_MAX_TITLE = {
+  en: 'max (sgd)',
+};
+const NEW_JOB_DURATION_TITLE = {
+  en: 'Job Duration',
+};
+const NEW_JOB_EXPIRY_TITLE = {
+  en: 'Job Expiry',
+};
+
 const NEW_JOB_TITLE_ID = 'new-job-title';
 const NEW_JOB_ADDRESS_ID = 'new-job-address';
 const NEW_JOB_DESCRIPTION_ID = 'new-job-description';
+const NEW_JOB_EXPIRY_ID = 'new-job-expiry';
+const NEW_JOB_DURATION_ID = 'new-job-duration';
+const REQUIREMENTS_LIST_NAME = 'requirements-list';
+const NEW_JOB_SUBMIT_ID = 'new-job-submit';
+const NEW_JOB_CANCEL_ID = 'new-job-cancel';
+
+const NEW_JOB_PAGE_TITLE_ID = 'new-job-page-title';
+const NEW_JOB_REQUIREMENTS_TITLE_ID = 'new-job-requirements-title';
+const NEW_JOB_PAY_TITLE_ID = 'new-job-pay-title';
+const NEW_JOB_DURATION_TITLE_ID = 'new-job-duration-title';
+const NEW_JOB_EXPIRY_TITLE_ID = 'new-job-expiry-title';
+
+const HOMEPAGE_PATH = '/walk-in-interview/src/main/webapp/index.html';
```

Comment:
are you sure the path is correct? i thought js path should be starting from the index not the src root